src/components/SettingsDialog/Participants/Participants.tsx

```diff
@@ -95,28 +117,35 @@ export const Participants = () => {
             .filter((participant) => participant.connected === onlineFilter)
             .map((participant) => (
               <li key={participant.user.id} className="participants__list-item">
-                <UserAvatar avatar={participant.user.avatar} className="participant__avatar" id={participant.user.id} ready={participant.ready} title={participant.user.name} />
-                <div className="participant__name-role-wrapper">
-                  <span className="participant__name">{participant.user.name}</span>
-                  {participant.role === "OWNER" || !isModerator || participant.user.id === self.user.id ? (
+                <UserAvatar
+                  avatar={participant.user.avatar}
+                  className={classNames("participant__avatar", {banned: participant.banned})}
+                  id={participant.user.id}
+                  ready={participant.ready}
+                  title={participant.user.name}
+                />
+                <div className={classNames("participant__name-role-wrapper", {banned: participant.banned})}>
+                  <span className={classNames("participant__name")}>{participant.user.name}</span>
+                  {(participant.role === "OWNER" || !isModerator || participant.user.id === self.user.id) && (
                     <span className="participant__role">
                       {participant.role === "OWNER" && t("UserRole.Owner")}
                       {participant.role === "MODERATOR" && t("UserRole.Moderator")}
                       {participant.role === "PARTICIPANT" && t("UserRole.Participant")}
                     </span>
-                  ) : (
+                  )}
+                  {participant.role !== "OWNER" && isModerator && participant.user.id !== self.user.id && !participant.banned && (
                     <div className="participant__role-buttons">
                       <button
                         className={classNames("participant__role", {"participant__role--active": participant.role === "MODERATOR"})}
-                        disabled={participant.role === "MODERATOR"}
+                        disabled={participant.role === "MODERATOR" || participant.banned}
```

Comment:
You don't need to disable them because the buttons are hidden if a participant is banned